shims/shims_for_IE.js

```diff
@@ -18,9 +18,9 @@ require('html5-history-api');  // HTML5 History: IE9.
 if (!Object.hasOwnProperty('name')) {
   Object.defineProperty(Function.prototype, 'name', {
     get: function getName() {
-      const matches = this.toString()
+      var matches = this.toString()
```

Comment:
Alternately if it's Typescript that's complaining, we could just make this file into typescript, which is probably as simple as changing the extension to `.ts`.